components/prow/manifests/nginx-controller.yaml

```diff
@@ -371,5 +371,5 @@ spec:
       paths:
       - backend:
           serviceName: grafana
-          servicePort: 80
+          servicePort: http
```

Comment:
This name should be `grafana-web` because that name is used in the [grafana-service](https://github.com/prometheus/prombench/blob/23d6d4e5930ee7f92706ecea6c97854ae66c398a/components/prombench/manifests/results/grafana_deployment.yaml#L88) definition